apps/nextra/pages/en/build/smart-contracts/book/SUMMARY.mdx

```diff
@@ -5,6 +5,10 @@
 - [Modules and Scripts](modules-and-scripts.mdx)
 - [Move Tutorial](move-tutorial.mdx)
 
+## Language Release Notes
```

Comment:
I don't see this section in the Left sidebar in the preview (https://developer-docs-nextra-9b9cy6461-aptoslabs.vercel.app/en/build/smart-contracts/book/SUMMARY).  Is something else needed to highlight the Release Notes and Move 2.0?

Reply:
Its under References at the bottom, which is good enough I think.

Reply:
Well, Up to you.  See gif I posted in the slack discussion.

Reply:
You need to use the newest Vercel link, I removed the one from the PR description because they seem to be changing.

Anyway I also move this up (before it was under 'References' because I misunderstood what is meant with 'References' (libraries). It is now right underneath Summary.